server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/handlers/GlobalExceptionHandler.java

```diff
@@ -73,4 +74,14 @@ public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundExcept
 
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(error);
     }
+
+    @ExceptionHandler(ValidationException.class)
+    public ResponseEntity<Object> handleValidationException(ValidationException ex) {
+        Map<String, String> error = new HashMap<>();
+        error.put(ERROR, ex.getMessage());
+
+        return ResponseEntity
+                .status(HttpStatus.BAD_REQUEST)
```

Comment:
Do you think this should be 409 CONFLICT instead of 400 BAD_REQUEST?

Reply:
Sure, updated